natscontext/context.go

```diff
@@ -492,7 +491,7 @@ func (c *Context) Save(name string) error {
 	}
 
 	if numCreds(c) > 1 {
-		return errors.New("too many types of credentials. Choose only one from 'user/password', 'creds', 'nkey', 'token', 'nsc'")
+		return errors.New("too many types of credentials. Choose only one from 'user/password/token', 'creds', 'nkey', 'nsc'")
```

Comment:
It's a bit confusing `user/password/token` suggests one of them but its not, I'd just say `user/token` since user+password kind of goes hand in hand as one

Reply:
I see we only validate on save which is awkward, maybe while you're here add a `Validate() error` function that does check and then instead call it here.  

In cli I want to validate and revert, but we cant validate via loading